src/app/shared/descriptor-type-compat.service.ts

```diff
@@ -14,7 +14,8 @@
  *     limitations under the License.
  */
 import { Injectable } from '@angular/core';
-import { ToolDescriptor } from './swagger';
+import { ToolDescriptor, Workflow } from './swagger';
+import DescriptorTypeEnum = Workflow.DescriptorTypeEnum;
 
 export enum WebserviceDescriptorTypeEnum {
```

Comment:
is this used anywhere anymore?

Reply:
Possibly not, I was more fixing the build so that tests would pass as opposed to making any improvements to take advantage of things.